Source/WebCore/rendering/style/TextSizeAdjustment.cpp

```diff
@@ -83,7 +83,7 @@ auto AutosizeStatus::computeStatus(const RenderStyle& style) -> AutosizeStatus
         if (lineHeight.isFixed() && lineHeight.value() - style.specifiedFontSize() > maximumDifferenceBetweenFixedLineHeightAndFontSize)
             return false;
 
-        if (style.whiteSpaceCollapse() == WhiteSpaceCollapse::Collapse && style.textWrap() == TextWrap::NoWrap)
+        if (style.whiteSpaceCollapse() == WhiteSpaceCollapse::Collapse && style.textWrapMode() == TextWrapMode::NoWrap)
```

Comment:
I really wish we had a boolean for `text-wrap-mode` instead of an enumeration for call sites like this.

Reply:
Same here